### PR TITLE
Feature/m2 1104 fill up calendar with real data

### PR DESCRIPTION
**Pull request by IhorNesterenko99**

### CODE REVIEW

File: src/modules/Dashboard/features/Applet/Schedule/Calendar/Calendar.styles.ts [hunk 1]
```diff
@@ -422,19 +409,15 @@ export const StyledCalendarWrapper = styled(Box, shouldForwardProp)`
...
-      }
-
       &:focus {
-        outline-color: ${variables.palette.primary};
+        outline: none;
       }
     }
 
+    .rbc-row-segment {
+      padding: ${theme.spacing(0, 0, 0.18, 0)};
```
Comment (muhurov) [inline src/modules/Dashboard/features/Applet/Schedule/Calendar/Calendar.styles.ts:418]: does it make sense having decimal part of pixel (1rem = 10px so 0.18rem = 1.8 px)?

File: src/modules/Dashboard/features/Applet/Schedule/Calendar/MonthWeekEvent/MonthWeekEvent.styles.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Box, styled } from '@mui/material';
+
+import { shouldForwardProp } from 'shared/utils';
+import { theme, variables } from 'shared/styles';
```
Comment (kdanilenka32) [inline src/modules/Dashboard/features/Applet/Schedule/Calendar/MonthWeekEvent/MonthWeekEvent.styles.ts:4]: space

File: src/modules/Dashboard/features/Applet/Schedule/Calendar/Calendar.styles.ts (hunk 1 above)
Comment (IhorNesterenko99) [reply src/modules/Dashboard/features/Applet/Schedule/Calendar/Calendar.styles.ts:418]: yes:)

File: src/modules/Dashboard/features/Applet/Schedule/Calendar/MonthWeekEvent/MonthWeekEvent.styles.ts (hunk 2 above)
Comment (IhorNesterenko99) [reply src/modules/Dashboard/features/Applet/Schedule/Calendar/MonthWeekEvent/MonthWeekEvent.styles.ts:4]: done

File: src/modules/Dashboard/features/Applet/Schedule/Calendar/MonthWeekEvent/MonthWeekEvent.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  const prevDaySlotEnd = subDays(slotEnd, 1);
+  const start = event.start > slotStart ? event.start : slotStart;
+  const end = event.end < prevDaySlotEnd ? event.end : prevDaySlotEnd;
+  const datesArray = end > start ? eachDayOfInterval({ start, end }) : [start];
+
+  return (
+    <StyledFlexTopCenter sx={{ height: '100%' }}>
+      {datesArray.map(() => (
+        <StyledEventWrapper
+          key={uniqueId()}
```
Comment (kdanilenka32) [inline src/modules/Dashboard/features/Applet/Schedule/Calendar/MonthWeekEvent/MonthWeekEvent.tsx:41]: not good solution to use uniqueId here

Comment (IhorNesterenko99) [reply src/modules/Dashboard/features/Applet/Schedule/Calendar/MonthWeekEvent/MonthWeekEvent.tsx]: done

**Pull request merged**